src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -42,33 +44,45 @@ public MauiMediaElement(MediaPlayerElement mediaPlayerElement)
 	{
 		this.mediaPlayerElement = mediaPlayerElement;
 
-		fullScreenButton = new Button
+		var fullScreenIcon = new FontIcon
 		{
-			Background = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
-			Foreground = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
-			Width = 40,
-			Height = 40
+			Glyph = "\uE740",
+			Foreground = new SolidColorBrush(Colors.White),
```

Comment:
Another suggestion I have is to not giving the button any colors to have its default colors which adapts itself with OS's dark and light mode just like Media Element transport controls